packages/graphql/src/translate/create-auth-and-params.ts

```diff
@@ -92,7 +93,7 @@ function createAuthPredicate({
                     res.params = { ...res.params, ...authPredicate[1] };
                 });
 
-                res.strs.push(`(${inner.join(` ${key} `)})`);
+                res.strs.push(joinPredicates(inner, key as "AND" | "OR"));
```

Comment:
To avoid casting the variable key, I think it is worth creating a new helper function :thinking: :
```ts
function isPredicateJoin(value: string): value is "AND" | "OR" {
    return ["AND", "OR"].includes(value)
}
```

And use this function instead of `if (key === "AND" || key === "OR") {`. This way key will be properly typed. Thoughts?

Reply:
Yes! Much better!